lib/aws-xray-sdk/model/metadata.rb

```diff
@@ -17,7 +17,11 @@ def sub_meta(namespace)
     end
 
     def to_h
-      @data
+      @data.keys.each_with_object({}) do |key, h|
```

Comment:
Same as above

Reply:
The same as above, we can't assume that developer using `cruby` or `jruby` to run their code, may be normal way to handle `to_h` is better.